subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -134,15 +134,36 @@ include::{samplesPath}/customPlugin/groovy/plugin/src/test/groovy/org/gradle/Gre
 
 With Gradle, it's very simple to implement a task that is skipped when all of its inputs and outputs are up to date (see <<more_about_tasks.adoc#sec:up_to_date_checks,Incremental Builds>>). However, there are times when only a few input files have changed since the last execution, and you'd like to avoid reprocessing all of the unchanged inputs. This can be particularly useful for a transformer task, that converts input files to output files on a 1:1 basis.
 
-If you'd like to optimise your build so that only out-of-date inputs are processed, you can do so with an _incremental task_.
+If you'd like to optimize your build so that only out-of-date inputs are processed, you can do so with an _incremental task_.
```

Comment:
```suggestion
If you'd like to optimize your build so that only out-of-date input files are reprocessed, you can do so with an _incremental task_.
```